deploy/sample-backup-configs/sample_config_build_recipes.yaml

```diff
@@ -14,8 +14,8 @@
 #    PLATFORM_CONFIG: Config
 #    deploy_triplet: null
 #    post_build_hook: null
-#    # REQUIRED: TODO: Will move to "run platform" work
-#    s3_bucket_name: firesim-AWSUSERNAME
+#    bit_builder:
+#      base_recipe: <>
```

Comment:
Will do... but to be clear this seems to be acceptable to people? I know in the past people found it annoying to add a bunch of extra lines to the recipes.

Reply:
I personally think that it would be nicer to have something of the form:
```
bit_builder_recipe: path
bit_builder_overrides: 
  <overrides>
```

Since in practice i don't suspect there will often be overrides provided. This cuts down one level of indentation and an extra line. 

However, this isn't consistent with the structure of other recipe+overrides sections, so i'm of two minds on the matter.

Reply:
Doing this but matching the other API a bit more by saying `bit_builder_arg_overrides`